### ISSUE TITLE
Add an accessibility statement

### ISSUE THREAD

**Issue by petros** [GitbookIO/public-docs#8]
This was triggered [by a discussion](https://gitbook.slack.com/archives/C014UCSK74Y/p1663745519668659?thread_ts=1663692767.114299&cid=C014UCSK74Y) in Slack on accessibility.

@BrettJephson pointed me to [Developing an Accessibility Statement | Web Accessibility Initiative (WAI) | W3C](https://www.w3.org/WAI/planning/statements/):

> This tool helps you create an accessibility statement for your own website, mobile application, or other digital content. You can download the statement you created, and further customize, style, and brand it. Read more below about accessibility statements or [generate an accessibility statement](https://www.w3.org/WAI/planning/statements/generator/#create) right away.

and I would love to explore adding something like that to our docs.

### Other ideas

- I wonder if we could create a custom integration (when the platform opens up) that allows our own customers to generate such a statement for their own website, mobile application, or other digital content from within their GitBook 😄 space itself.
